fhir-persistence/src/main/java/com/ibm/fhir/persistence/helper/FHIRTransactionHelper.java

```diff
@@ -38,12 +38,13 @@ public void begin() throws FHIRPersistenceException {
                 txn.begin();
                 txnStarted = true;
             } else {
-                log.fine("Transaction is already active on current thread...");
+                log.fine("Transaction is already active on current thread, get Db connection only ...");
+                txn.begin();
```

Comment:
I don't think this is the right approach.

Reply:
your suggestion?

Reply:
I think you have unintended consequences.  I'm not offering a suggestion here.  I think John should review